### PR TITLE
Feature/43 ranges

### PR DESCRIPTION
**Pull request by blabadi**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:8
+FROM node:12.5.0-alpine
```
Comment (blabadi) [inline Dockerfile]: i think we should move from node 8 since it's reaching it's LTS end

File: src/config/MetaSchema.json [hunk 2]
```diff
@@ -65,19 +81,195 @@
...
+                        {
+                            "required": [
+                                "min"
+                            ],
+                            "properties": {
+                                "min": {
+                                    "type": "integer"
+                                },
+                                "exclusiveMin": {
+                                    "not": {}
```
Comment (blabadi) [inline src/config/MetaSchema.json]: maybe i don't need the not if I use `additionalProperties : false`, will test that

File: Dockerfile (hunk 1 above)
Comment (andricDu) [reply Dockerfile]: :+1:

File: src/config/MetaSchema.json [hunk 3]
```diff
@@ -65,19 +81,195 @@
...
+                            ],
+                            "properties": {
+                                "min": {
+                                    "not": {}
+                                },
+                                "exclusiveMin": {
+                                    "not": {}
+                                },
+                                "max": {
+                                    "not": {}
```
Comment (andricDu) [inline src/config/MetaSchema.json]: why the `not`?

**Pull request merged**